namd/src/GlobalMasterColvars.h

```diff
@@ -0,0 +1,26 @@
+#ifndef GLOBALMASTERCOLVARS_H
+#define GLOBALMASTERCOLVARS_H
+
+#include <memory>
+
+#include "colvarmodule.h"
+#include "colvarproxy.h"
+#include "colvarproxy_namd.h"
+
+
+class GlobalMasterColvars : public GlobalMaster {
+public:
+
+  GlobalMasterColvars() : cp(new colvarproxy_namd(this)) {}
+
+  void calculate() override {
+    cp->calculate();
```

Comment:
Maybe use more explicit identifiers than `cp`, and most importantly, `gm_`. This makes the code more hermetic.

For `cp`, why not `proxy`, and for `gm_`, maybe just `globalmaster`? The more obvious, the better.